knimin/handlers/ag_add_barcode_kit.py

```diff
@@ -7,11 +7,13 @@
 class AGAddBarcodeKitHandler(BaseHandler):
```

Comment:
would it be possible to have the methods default to a high access level? In other words, unless explicitly stated, you need `Admin`. It's conservative in the event some handlers get over looked, or future development forgets to add the `self.has_access` calls